docs/_templates/layout.html

```diff
@@ -210,7 +216,53 @@
           {% else %}
           <div class="rst-content">
             {% endif %}
-            {% include "breadcrumbs.html" %}
+
+            <!-- Replace the RTD searchbar with Algolia DocSearch -->
+            <div style="float: right;">
+              <input type="text" class="input" id="docsearch-input" placeholder="Search docs"/>
+            </div>
+
+            <!-- Insert breadcrumb (since _templates/breadcrumbs.html doesn't seem to import correctly) -->
```

Comment:
@randyzwitch Not really sure why breadcrumbs.html doesn't import (aka any code I put into there doesn't show up in the docs), but I guess we can just delete that file entirely after this change?

Reply:
No idea, but I suspect it's part of the default theme and not something I put. And would be inclined to keep it, since I don't know it's history.

Reply:
Okay! Happy to live with the code duplication in that case.